src/rest.rs

```diff
@@ -957,6 +957,38 @@ fn handle_request(
 
             json_response(prepare_txs(txs, query, config), TTL_SHORT)
         }
+        (
+            &Method::GET,
+            Some(script_type @ &"address"),
+            Some(script_str),
+            Some(&"txs"),
+            Some(&"summary"),
+            last_seen_txid,
+        )
+        | (
+            &Method::GET,
+            Some(script_type @ &"scripthash"),
+            Some(script_str),
+            Some(&"txs"),
+            Some(&"summary"),
+            last_seen_txid,
+        ) => {
+            let script_hash = to_scripthash(script_type, script_str, config.network_type)?;
+            let last_seen_txid = last_seen_txid.and_then(|txid| Txid::from_hex(txid).ok());
+            let max_txs = cmp::max(
+                config.rest_default_max_address_summary_txs,
+                query_params
```

Comment:
why would this be max?

That means that people calling the API can set the limit higher but not lower.

min?

Reply:
yep, typo, should be min 😅